src/Lykke.AlgoStore.CSharp.AlgoTemplate.Services/Services/WorkflowService.cs

```diff
@@ -221,29 +225,36 @@ private ICandleContext CreateCandleContext(IAlgoCandle candle)
         private void SetContextProperties(Context context)
         {
             context.Functions = _functionsService.GetFunctionResults();
-
-            context.Actions = actions;
+            //context.Actions = actions;
         }
 
         public void SetUpAlgoParameters(AlgoClientInstanceData algoInstance)
         {
-            if (algoInstance == null || algoInstance.AlgoMetaDataInformation.Parameters == null)
-                return;
-
-            Type parameterType = _algo.GetType();
-
-            foreach (var parameter in algoInstance.AlgoMetaDataInformation.Parameters)
+            try
             {
-                PropertyInfo prop = parameterType.GetProperty(parameter.Key);
+                if (algoInstance == null || algoInstance.AlgoMetaDataInformation.Parameters == null)
+                    return;
 
-                if (prop != null && prop.CanWrite)
+                Type parameterType = _algo.GetType();
+
+                foreach (var parameter in algoInstance.AlgoMetaDataInformation.Parameters)
                 {
-                    if (prop.PropertyType.IsEnum)
-                        prop.SetValue(_algo, Enum.ToObject(prop.PropertyType, Convert.ToInt32(parameter.Value)), null);
-                    else
-                        prop.SetValue(_algo, Convert.ChangeType(parameter.Value, prop.PropertyType), null);
+                    PropertyInfo prop = parameterType.GetProperty(parameter.Key);
+
+                    if (prop != null && prop.CanWrite)
+                    {
+                        if (prop.PropertyType.IsEnum)
+                            prop.SetValue(_algo, Enum.ToObject(prop.PropertyType, Convert.ToInt32(parameter.Value)), null);
+                        else
+                            prop.SetValue(_algo, Convert.ChangeType(parameter.Value, prop.PropertyType), null);
+                    }
                 }
             }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```

Comment:
Should we output error to console (it will be visible in Kube pod log)?
Maybe we should log this into our (custom) user log or in regular log.

Reply:
This is something that I used for tests I will remove it, thanks for the remark!